test/integration/merkle-lockup/lt/get-tranches-with-percentage/getTranchesWithPercentage.t.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.22 <0.9.0;
+
+import { MerkleLockupLT } from "src/types/DataTypes.sol";
+
+import { MerkleLockup_Integration_Test } from "../../MerkleLockup.t.sol";
+
+contract GetTranchesWithPercentages_Integration_Test is MerkleLockup_Integration_Test {
```

Comment:
We dont need this test. We are already testing `getTranchesWithPercentages` in `Constructor_MerkleLockupLT_Integration_Test`